.scalafmt.conf

```diff
@@ -1,16 +1,19 @@
 version = "3.7.2"
+runner.dialect = scala213
 maxColumn = 120
-align = most
+align.preset = most
 continuationIndent.defnSite = 2
 assumeStandardLibraryStripMargin = true
-docstrings = JavaDoc
+docstrings.style = Asterisk
 lineEndings = preserve
 includeCurlyBraceInSelectChains = false
-danglingParentheses = true
+danglingParentheses.preset = true
 optIn.annotationNewlines = true
 
 rewrite.rules = [RedundantBraces]
 project.excludeFilters = [
   "core/src/main/scala/zio/config/ProductBuilder.scala",
-  "core/src/test/scala/zio/config/ProductBuilderTest.scala"
+  "core/src/test/scala/zio/config/ProductBuilderTest.scala",
+  "core/shared/src/main/scala-3.x/*",
+  "magnolia/shared/src/main/scala-dotty/*"
```

Comment:
I think this can also go into a custom module within ecosystem plugin that wraps scalafmt
And for custom settings, such as exclude, I could simply configure with in each module setting, `excludeFmt`

Reply:
Thank you! Added to the backlog.